blocks/footer/footer.css

```diff
@@ -1,14 +1,91 @@
-footer {
-  padding: 2rem;
-  background-color: var(--light-color);
-  font-size: var(--body-font-size-s);
-}
-
 footer .footer {
-  max-width: 1200px;
+  max-width: 100vw;
   margin: auto;
+  background-color: #1a355e;
+  color: #fff;
```

Comment:
color: white or color: var(--background-color) would be preferred here.  It seems pedantic but realisically, if they change their entire color scheme it makes it a lot more convenient to have everything based on variables.